src/bun.js/node/node_os.zig

```diff
@@ -0,0 +1,388 @@
+
+            var list = map.get(interface) orelse std.ArrayList(JSC.JSValue).init(heap_allocator);
+            var obj = JSC.JSValue.createEmptyObject(globalThis, if (std.mem.eql(u8, family, "IPv6")) 7 else 6);
+            obj.put(globalThis, &JSC.ZigString.init("address"), JSC.ZigString.init(std.mem.span(part.address)).withEncoding().toValueGC(globalThis));
+            obj.put(globalThis, &JSC.ZigString.init("netmask"), JSC.ZigString.init(netmask).withEncoding().toValueGC(globalThis));
+            obj.put(globalThis, &JSC.ZigString.init("family"), JSC.ZigString.init(family).withEncoding().toValueGC(globalThis));
+            obj.put(globalThis, &JSC.ZigString.init("mac"), JSC.ZigString.init(std.mem.span(part.mac)).withEncoding().toValueGC(globalThis));
+            obj.put(globalThis, &JSC.ZigString.init("cidr"), JSC.ZigString.init(cidr).withEncoding().toValueGC(globalThis));
+            if (std.mem.eql(u8, family, "IPv6")) obj.put(globalThis, &JSC.ZigString.init("scopeid"), JSC.JSValue.jsNumber(part.scopeid));
+            obj.put(globalThis, &JSC.ZigString.init("internal"), JSC.JSValue.jsBoolean(if (part.internal == 0) true else false));
+
+            _ = list.append(obj) catch unreachable;
+            _ = map.put(interface, list) catch unreachable;
+        }
+
+        for (map.keys()) |key| {
+            var value = map.get(key);
+
+            object.put(globalThis, &JSC.ZigString.init(key), JSC.JSArray.from(globalThis, value.?.toOwnedSlice()));
+        }
+
+        return object;
+    }
+
+    pub fn platform(globalThis: *JSC.JSGlobalObject, _: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+        if (comptime is_bindgen) return JSC.JSValue.jsUndefined();
+
+        return JSC.ZigString.init(Global.os_name).withEncoding().toValueGC(globalThis);
+    }
+
+    pub fn release(globalThis: *JSC.JSGlobalObject, _: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+        if (comptime is_bindgen) return JSC.JSValue.jsUndefined();
+
+        return JSC.ZigString.init(C.getRelease()).withEncoding().toValueGC(globalThis);
+    }
+
+    pub fn setPriority(globalThis: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+        if (comptime is_bindgen) return JSC.JSValue.jsUndefined();
+
+        var args_ = callframe.arguments(2);
+        var arguments: []const JSC.JSValue = args_.ptr[0..args_.len];
+
+        if (arguments.len == 0) {
+            const err = JSC.toTypeError(
+                JSC.Node.ErrorCode.ERR_INVALID_ARG_TYPE,
+                "The \"priority\" argument must be of type number. Received undefined",
+                .{},
+                globalThis,
+            );
+            globalThis.vm().throwError(globalThis, err);
+            return JSC.JSValue.jsUndefined();
+        }
+
+        var pid = if (arguments.len == 2) arguments[0].asInt32() else 0;
+        var priority = if (arguments.len == 2) arguments[1].asInt32() else arguments[0].asInt32();
```

Comment:
```suggestion
        const priority = if (arguments.len == 2) arguments[1].toInt32(globalThis) else arguments[0].toInt32(globalThis);
```

Reply:
`asInt32` says "cast this as an int32"

`toInt32(global)` says "convert this into an int32 even if it isn't currently"